superset-frontend/src/setup/setupColors.js

```diff
@@ -49,4 +50,6 @@ export default function setupColors() {
     });
   });
   sequentialSchemeRegistry.setDefaultKey('superset_seq_1');
+
+  setupColorsExtra();
```

Comment:
Should we pass the context here as in `setupColorsExtra({sequentialSchemeRegistry, categoricalSchemeRegistry});` or similar?

I'm also wondering if we should have a central file for all the configuration hooks, similar to `superset/config.py` on the python side?

Reply:
hi @mistercrunch I really liked your idea and added another commit  supporting color schemes from the config.py file